robyn_rate_limits/protocols.py

```diff
@@ -2,5 +2,8 @@
 
 
 class LimitStore(Protocol):
-    def get_calls_count(self, limit_key: str, current_timestamp: int) -> int:
```

Comment:
Can you please revert styling changes? I see many places where the change is adding a trailing command or removing it which changes the way `ruff` formats the files. Please let it determine the line length by itself and don't change it by manipulating trailing commas.